src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -770,9 +774,43 @@ public void triggerComments() {
     scrollToFooter();
   }
 
-  public void scrollToSlot(String slot) {
-    WebElement slotSelector = driver.findElement(By.cssSelector(slot));
-    jsActions.scrollToElement(slotSelector);
-    PageObjectLogging.log("scrollToSlot", "Scroll to the slot " + slot, true);
+  private WebElement pageSelector(String webSelector) {
```

Comment:
Which one is more readable for you?
A) `jsActions.scrollToSpecificElement(pageSelector(selector));`
B) `jsActions.scrollToSpecificElement(driver.findElement(By.cssSelector(selector)));`